doc/sphinx/source/developer_guide/abi.rst

```diff
@@ -49,58 +82,68 @@ Include header and declare an ``abi_event`` as a global ``static`` variable (but
      // do something here
     }
 
+.. warning:: Make sure the callback doesn't take too much time (process the received data elsewhere), as ABI callbacks are not thread safe.
+
 In the initialization function (or later) call the binding function for the message you want to receive.
 
 .. code-block:: C
 
     AbiBindMsgDATA(ABI_BROADCAST, &ev, data_cb);
 
-
 The first parameter is the sender ID you want to receive the message from.
+Senders IDs can be found in the file ``sw/airborne/modules/core/abi_sender_ids.h``
```

Comment:
I think we should keep a note here to recall the special binding IDs `ABI_DISABLE` and `ABI_BROADCAST`